lib/web_ui/lib/src/engine/window.dart

```diff
@@ -689,6 +689,46 @@ class EngineWindow extends ui.Window {
     // callback(null);
   }
 
+  bool _isHistoryInitialized = false;
+
+  @visibleForTesting
```

Comment:
`@visibleForTesting` in the engine code doesn't mean much because we're one giant library where everything has access to everything else (even private methods), whether you have `@visibleForTesting` or not.

Reply:
To me at least, it just indicates the purpose of the field. Similar to how we still use private members (even though it doesn't mean anything in the engine). It's useful when I look at the code to be able to immediately tell: Oh this is meant to be private, or this is meant to only be accessed by tests.